color/src/rgba8.rs

```diff
@@ -24,8 +24,49 @@ pub struct Rgba8 {
     pub a: u8,
 }
 
+impl Rgba8 {
+    /// Returns the color as a packed value.
```

Comment:
It would be good to document that `r` is the most significant byte and `a` the least. Also I think this can and should be const.